MonkeyChat/ConversationListViewController.swift

```diff
@@ -343,7 +346,8 @@ class ConversationsListViewController: UITableViewController {
     }
     
     for user in DBManager.getUsers(conversation.members as NSArray as! [String]) {
```

Comment:
Let's move this `for` inside `ChatViewController` instead of line 91: `self.nameMembersDescription = (self.nameMembers as NSArray).componentsJoined(by: ", ")`